ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/EthSignerBaseCommand.java

```diff
@@ -107,6 +107,15 @@
       arity = "1")
   private String downstreamHttpHost = InetAddress.getLoopbackAddress().getHostAddress();
 
+  @SuppressWarnings("FieldMayBeFinal")
+  @Option(
+      names = {"--downstream-http-request-args"},
```

Comment:
this is probably more like the downstream-http-path? I.e. its not technically part of the request, rather its part of the address

Reply:
This is a good point, I will change to http path instead that makes more sense